pom.xml

```diff
@@ -17,17 +17,13 @@
             <id>spigot-repo</id>
             <url>https://hub.spigotmc.org/nexus/content/repositories/snapshots/</url>
         </repository>
-        <repository>
-            <id>sonatype</id>
-            <url>https://oss.sonatype.org/content/groups/public/</url>
-        </repository>
     </repositories>
     <dependencies>
         <!--Bukkit API-->
         <dependency>
             <groupId>org.spigotmc</groupId>
             <artifactId>spigot-api</artifactId>
-            <version>1.21.5-R0.1-SNAPSHOT</version>
+            <version>1.21.8-R0.1-SNAPSHOT</version>
```

Comment:
Make sure this is approved. I don't think you're doing anything 1.21.8 specific or anything but just make sure we're aligning with the version ProjectKorra is using. We don't want to in theory have ProjectKorraRPG require a different version than base PK (Ik this isn't the case rn). Verify that PK doesn't have any issues with later versions being sorted out and that it's been confirmed PK will be at this version then update this to match. 

If you already did this feel free to resolve.

Reply:
Just match what PK is using which iirc is 1.21.4. Remember that we're on the drop  release schedule so there are very significant changes between "minor versions" now lol. We should be able to leverage everything available in the version PK uses